src/windows/export.py

```diff
@@ -265,6 +265,10 @@ def __init__(self):
         # Determine the length of the timeline (in frames)
         self.updateFrameRate()
 
+        # Tooltips
+        self.lblVideoBitrate.setToolTip(_("Type <b>N crf</b><br> where N = 0..63, to use Constant Rate Factor control for quality setting instead of the bitrate"))
+        self.txtVideoBitRate.setToolTip(_("Type <b>N crf</b><br> where N = 0..63, to use Constant Rate Factor control for quality setting instead of the bitrate"))
```

Comment:
If _nothing_ else, please change this to:
```python
msg = _("Incredibly long string I'd be crazy to write twice.")
self.lblVideoBitrate.SetToolTip(msg)
self.txtVideoBitrate.SetToolTip(msg)
```
...Why would you _do_ that to yourself?? I'm sure it was a copy-paste, but still, what if it had to be edited? :laughing:

Reply:
> ...what if it had to be edited?

Same copy/paste rule would apply.

But OK for convenience I'll reformat this.